### ISSUE TITLE
Accessibility Bug: Feature toggle for claimant search

### ISSUE THREAD

**Issue by candilemoine** [department-of-veterans-affairs/va.gov-team#109937]
### Description
[QA Standards Dashboard](https://testing-tools-testing-reports.s3-us-gov-west-1.amazonaws.com/vets-website-cypress-reports/be1ea81b-a3f6-4741-95f0-23fb86b87cf2.html) is flagging an accessibility violation. 

```
AssertionError: 1 accessibility violation was detected: expected 1 to equal 0

Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `App feature toggle and Pilo...`
    at Context.eval (http://localhost:3001/__cypress/tests?p=src/platform/testing/e2e/cypress/support/index.js:19559:18)
cy.contains("Search people").should("be.visible");
``` 

### Task
- resolve violation

**Comment by jquispe-oddball** [comment]
@candilemoine I was able to update my axe test locally and recreate issue

<img width="824" alt="Image" src="https://github.com/user-attachments/assets/c49ed3c1-469f-467a-86d3-8085895f3959" />

Fixed issue and cleared out all errors

<img width="801" alt="Image" src="https://github.com/user-attachments/assets/26bcd710-0eaa-4721-8b3f-15099ed235d4" />

**Comment by candilemoine** [comment]
@jquispe-oddball is this just waiting for Platform review before going to production?
